content/en/serverless/guide/agent_configuration.md

```diff
@@ -12,7 +12,7 @@ The Agent's [main configuration file][1] is `datadog.yaml`. For the serverless A
 |--------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `DD_API_KEY`                   | The environment variable with your Datadog API key, in plaintext. **One** key environment variable is required. See [serverless CLI environment variables][7].                                                                                              |
 | `DD_KMS_API_KEY`               | The environment variable with your Datadog API key, using KMS. **One** key environment variable is required. See [serverless CLI environment variables][7].                                                                        |
-| `DD_API_KEY_SECRET_ARN`        | The environment variable with your Datadog API key, using an secret manager **One** key environment variable is required. See [serverless CLI environment variables][7].                                                                           |
+| `DD_API_KEY_SECRET_ARN`        | The environment variable with your Datadog API key, using a secret manager. **One** key environment variable is required. See [serverless CLI environment variables][7].                                                                           |
```

Comment:
sorry this should be a proper noun, AWS Secrets Manager is the specific product we reference here: https://aws.amazon.com/secrets-manager/